scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -261,9 +261,11 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
                 </tr>
               </thead>
               <tbody>
+                 {% set audit_query = namespace() %}
                  {% for audit in audits %}
+                  {% set audit_query = audit.link|url_args %}
                   <tr>
-                    <td><strong>{{ audit.subject }}</strong> was marked checked by {{ audit.user_name }} on {{audit.created_at.strftime('%Y-%m-%d')}}.</td>
+                    <td><strong>{{ audit.subject }} (category:{{ audit_query.category if audit_query.category else "NA"}}, type:{{audit_query.variant_type if audit_query.variant_type else "NA"}})</strong> was marked checked by {{ audit.user_name }} on {{audit.created_at.strftime('%Y-%m-%d')}}.</td>
```

Comment:
```suggestion
                    <td><strong>{{ audit.subject }} ({{audit_query.variant_type if audit_query.variant_type else "type unavailable -"}} {{ audit_query.category if audit_query.category else "category unavailable"}})</strong> was marked checked by {{ audit.user_name }} on {{audit.created_at.strftime('%Y-%m-%d')}}.</td>
```
Lets save some ink - stuff like "clinical SV" is not really made more clear with "category" and "type" added.